docs/references/express/overview.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: Clerk Express SDK
+description: Learn how to integrate Express into your Clerk application.
+---
+
+Clerk makes it simple to add authentication to your Express application. This documentation covers the capabilities and methods available from Clerk's Express SDK.
+
+See the [quickstart](/docs/quickstarts/express) to get started.
+
+## Available methods
+
+### `clerkMiddleware()`
+
+The `clerkMiddleware()` function checks the request's cookies and headers for a session JWT and, if found, attaches the [`Auth`](/docs/references/nextjs/auth-object#auth-object) object to the request object under the `auth` key.
+
+```js
+import { clerkMiddleware } from '@clerk/express'
+
+const app = express()
+
+// Pass no parameters
+app.use(clerkMiddleware())
+
+// Pass options
+app.use(clerkMiddleware(options))
+```
+
+#### Options
+
+The `clerkMiddleware()` function accepts [these options](https://clerk.com/docs/references/nextjs/clerk-middleware#clerk-middleware-options) plus the following:
+
+<Properties>
+  - `clerkClient`
+  - [`ClerkClient`](/docs/references/backend/overview#create-clerk-client-options)
+
+  An instance of the ClerkClient class. This is used to interact with the Clerk API.
+
+  ---
+
+  - `debug`
+  - `boolean`
+
+  A flag to enable debug mode. When enabled, the middleware will log debug information to the console.
+
+  ---
+
+  - `enableHandshake`
+  - `boolean`
+
+  Enables Clerk's handshake flow, which helps verify the session state when a session JWT has expired. It issues a 307 redirect to refresh the session JWT if the user is still logged in.
+</Properties>
+
+### `requireAuth`
+
+The `requireAuth()` middleware functions similarly to `clerkMiddleware()`, but also protects your routes by redirecting unauthenticated users to the sign-in page.
```

Comment:
```suggestion
The `requireAuth()` middleware functions similarly to `clerkMiddleware()`, but also protects your routes by redirecting unauthenticated users to the sign-in page. It will look for a `CLERK_SIGN_IN_URL` environment variable and use that by default. If not present, you will need to pass a sign in url to the middleware as an option.
```